bundles/org.openhab.binding.tacmi/src/main/java/org/openhab/binding/tacmi/internal/json/obj/Value.java

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
```

Comment:
Please update all headers for new files to contain 2025
```suggestion
 * Copyright (c) 2010-2025 Contributors to the openHAB project
```

Reply:
Working on that. Forgot to run verify. :-(